frontend/views/pages/PayGroup.vue

```diff
@@ -128,14 +107,8 @@ export default {
   },
   data () {
     return {
-      ephemeral: {
-        admirations: {
-          types: [this.L('Awesome'), this.L('Cool'), this.L('Great'), this.L('Nice'), this.L('Super'), this.L('Sweet')],
-          users: [] /* ['admiration'] */
-        }
-      },
       fakeStore: {
-        users: [
+        usersToPay: [
```

Comment:
@mmbotelho you can tweak the values of these users to mock some state you might want to review.

Reply:
Thank you for leaving me these tips & tricks, it's very useful!